timemachine-compose-smb-arm7l.yml

```diff
@@ -21,7 +21,8 @@ services:
       - SET_PERMISSIONS=false
       - SHARE_NAME=TimeMachine
       - SMB_INHERIT_PERMISSIONS=no
-      - SMB_NFS_ACES=yes
+      - SMB_NFS_ACES=no
```

Comment:
Just curious why you changed this setting.  The idea of the docker-compose files is to show the default values to allow them to be tweaked.

Reply:
I had one mac that would immediately fail when starting a new TM backup from scratch with the default setting of `yes`. I believe it was from a permissions issue. With the setting of `no`, it allowed the initial backup to start. So `no` seemed to be a better option.

But I am not opposed to leaving it set to the default of `yes` for consistency. I have my own local copy of the compose file with other superficial tweaks for running locally so it is not a big inconvenience for me. I will update the PR.

Reply:
Just curious - any chance that it was related to https://github.com/mbentley/docker-timemachine/issues/69 where the `SMB_VFS_OBJECTS` default values were recently changed?  In  that instance, it was specifically not working for new backups without `acl_xattr`

Reply:
Luckily I didn't find and start using this project until after this issue was found and fixed. So `acl_xattr` was included in all my testing. I have two Macs, both on Big Sur. One worked fine and one had the issue. I still can't fully explain why.